book/http.md

```diff
@@ -141,7 +141,7 @@ Requesting Information
 Once it's connected, the browser requests information from the server
 by giving its *path*, the path being the part of a URL that comes
 after the host name, like `/index.html`. The structure of the request
-is shown in Figure 2. You can type this into `telnet` to try it.
+is shown in Figure 2. Type this into `telnet` to try it.
```

Comment:
Later on, we use the imperative (line 182 in the source, "So type a blank line into `telnet`") so I think we should use it here too.